src/components/skip-link/SkipLink.stories.mdx

```diff
@@ -1,15 +1,18 @@
 import {ArgsTable, Meta, Story, Canvas} from '@storybook/addon-docs';
-import {within} from '@storybook/testing-library';
+import {within, screen, userEvent} from '@storybook/testing-library';
 
 import SkipLink from './SkipLink';
 import SkipLinkA11y from './stories/SkipLink.a11y.mdx';
+function sleep(ms) {
+  return new Promise(resolve => setTimeout(resolve, ms));
+}
 
 export const play = async ({canvasElement}) => {
   const canvas = within(canvasElement);
   const link = canvas.getByRole('link', {
     name: /skip to main content/i,
   });
-  link.focus();
+  await sleep(300);
```

Comment:
@katarzynatobis i think you can use `  chromatic: { delay: 300 },` in chromatic story instead

Reply:
I tried this custom function and chromatic delay - both ways did not work (for some reason css animation is not played). Finally, I just removed from snapshot the animation of focus ring.

Reply:
cool let's use that